.github/scripts/spacecheck.pl

```diff
@@ -27,25 +27,16 @@
 use warnings;
 
 my @tabs = (
-    "^m4/zz40-xc-ovr.m4",
-    "Makefile\\.(am|example)\$",
-    "/mkfile",
-    "\\.(bat|sln|vc)\$",
+    "^m4/zz40-xc-ovr.m4", "Makefile\\.(am|example)\$",
+    "/mkfile", "\\.(bat|sln|vc)\$",
```

Comment:
This was a laid out intentinally like this for readability.
After reformatting, it's bit a of a mess.

Is there a way to disable it reflowing lists? If not, perhaps it should be recoded differently.

Reply:
probably, perltidy has many switches and learning what they do takes time - I appreciate all the 'eyeballs' on this as I would not on my own be able to grok everything.